ix-dev/community/windmill/ix_values.yaml

```diff
@@ -0,0 +1,29 @@
+images:
+  image:
+    repository: ghcr.io/windmill-labs/windmill
+    tag: "1.485.3"
+  postgres_16_image:
+    repository: postgres
+    tag: "16"
+  caddy_image:
+    repository: ghcr.io/windmill-labs/caddy-l4
+    tag: "latest"
```

Comment:
please use a pinned version here.

Reply:
there is no version number here. is it appropriate to use a hash?

Reply:
lets leave it as latest then for now